stagehand x mcp-use/app.py

```
@@ -0,0 +1,201 @@
+
+
+
+def handle_load_sample():
+    st.session_state.config_json = SAMPLE_CONFIG_STR
+
+def handle_activate():
+    if st.session_state.client is not None:
+        try:
+            asyncio.run(st.session_state.client.close_all_sessions())
+        except Exception:
+            pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)